src/lib/crosslink.ts

```diff
@@ -34,11 +33,8 @@ export function generateCrosslink(code: string, options?: LinkOptions) {
 
   if (options && options.targetPlatform) {
     const target =
-      options.targetPlatform && options.targetPlatform.toUpperCase();
-
-    if (!isKeyOfObject(target, TargetPlatforms))
-      throw new Error('Invalid target platform');
-
+      options.targetPlatform && getTargetById(options.targetPlatform);
```

Comment:
It's real bug - it's impossible to pass `js-ir` platform. Cause expected `js_ir`.
Test below.